README.md

```diff
@@ -4,10 +4,20 @@ Omnitruck API service that provides license validation and entitlement checking
 
 ## Requirements
 
-* golang 1.19+
+- golang 1.19+
 
 ## Getting Started
 
+For getting the automate, habitat and related products information
+
+```bash
+ACCESS_KEY="AWS access key"
+SECRET_KEY="AWS secret key"
+REGION="AWS region in which Dynamodb database is present"
+PRODUCT_TABLE_NAME="To get the information regarding Automate and Habitat details"
+RELATED_PRODUCT_TABLE_NAME="To get information ragarding the relatedProducts"
```

Comment:
```suggestion
RELATED_PRODUCTS_TABLE_NAME="To get information ragarding the relatedProducts"
```
this is correct variable name

Reply:
Uploaded the spelling error for this as suggested